pkg/webservice/handlers.go

```diff
@@ -290,6 +293,20 @@ func getApplicationJSON(app *objects.Application) *dao.ApplicationDAOInfo {
 			Partition:        alloc.PartitionName,
 		}
 		allocationInfos = append(allocationInfos, allocInfo)
+		if alloc.IsPlaceholder() {
```

Comment:
This would add a lot of overhead: an application with 1000 placeholders in the same task group would do this 1000 times, it only needs to be done once.
The placeholder data we display is a top level object on the application, not one for each allocation

Reply:
Something like this:
```
	placeholders := app.GetPlaceholderData()
	placeholderInfos := make([]dao.PlaceholderDAOInfo, 0, len(placeholders))
	for _, taskGroup :=  range placeholders {
		...
		placeholderInfos = append(placeholderInfos, placeholderInfo)
	}
```